openpgp-dsm/src/lib.rs

```diff
@@ -884,6 +885,119 @@ pub fn generate_key(
     Ok(())
 }
 
+pub struct DsmKeyInfo {
+    name: String,
+    kid: Uuid,
+    hashing_algo: Option<HashAlgorithm>,
```

Comment:
Hashing algorithm? Why?
We should be showing the crypto algorithm of the primary key. E.g. RSA (bitsize) or EC (curve)

Reply:
When I run the test script I get:
```
generate-knownkeys-test-IMW0VcxkpY-cv25519:
    UUID: c0f31763-5c4f-45d9-85ab-6d05aeb58c45
    Hashing algorithm: None
    Created at: 2023-02-20 21:11:10 UTC
    Last used at: NA
    PGP fingerprint: 17D0999363E48109BE2003CC2E864D5120879189
```